code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -323,8 +423,61 @@
 	cardigan["cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan
 	cardigan["sweater cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/sweater
 	cardigan["argyle cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/argyle
+	cardigan["crop top cardigan"] = /obj/item/clothing/suit/storage/toggle/cardigan/croptop
 	gear_tweaks += new/datum/gear_tweak/path(cardigan)
 
+/datum/gear/suit/denim_jacket
+	display_name = "denim jacket (selection)"
+	description = "A selection of denim jackets."
+	path = /obj/item/clothing/suit/storage/toggle/denim_jacket
+
+/datum/gear/suit/denim_jacket/New()
+	..()
+	var/denimjackets = list()
+	denimjackets["denim jacket"] = /obj/item/clothing/suit/storage/toggle/denim_jacket
+	denimjackets["denim jacket, corporate"] = /obj/item/clothing/suit/storage/toggle/denim_jacket/nanotrasen
+	denimjackets["denim vest"] = /obj/item/clothing/suit/storage/toggle/denim_jacket/sleeveless
+	denimjackets["denim vest, corporate"] = /obj/item/clothing/suit/storage/toggle/denim_jacket/nanotrasen/sleeveless
+	gear_tweaks += new/datum/gear_tweak/path(denimjackets)
+
+/datum/gear/suit/greatcoat
+	display_name = "greatcoat"
+	path = /obj/item/clothing/suit/storage/toggle/greatcoat
+
+/datum/gear/suit/snowsuit
+	display_name = "snowsuit"
```

Comment:
Remove the snowsuits. There is no real reason for NT to supply people with those.

Reply:
Except they supply people with winter coats and pairs of winter boots. Departmental ones, to be exact. Might I ask what are the reasons for those?